helix-core/src/test/java/org/apache/helix/manager/zk/TestRawZkClient.java

```diff
@@ -91,29 +94,137 @@ void testGetStat() {
     AssertJUnit.assertNotSame(stat, newStat);
   }
 
-  @Test()
-  void testSessionExpire() throws Exception {
-    IZkStateListener listener = new IZkStateListener() {
+  /*
+   * Tests subscribing state changes for helix's IZkStateListener.
+   */
+  @Test
+  public void testSubscribeStateChanges() {
+    int numListeners = _zkClient.numberOfListeners();
```

Comment:
My thought was, since _zkClient is a global var in the parent class. I can't assume number of listeners is 0. There might already be some listeners in the _zkClient. So I just get the number, instead of 0.